work_items/kyc-vcs/schemas/kyc-credential.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "KYC Credential Schema",
+  "description": "Schema for a KYC Credential",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "object",
+      "properties": {
+        "familyName": {
+          "type": "string"
+        },
+        "givenName": {
+          "type": "string"
+        },
+        "additionalName": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "birthDate": {
+      "type": "string"
+    },
+    "postalAddress": {
+      "type": "object",
+      "properties": {
+        "addressCountry": {
+          "type": "string"
+        },
+        "addressLocality": {
+          "type": "string"
+        },
+        "addressRegion": {
+          "type": "string"
+        },
+        "postOfficeBoxNumber": {
+          "type": "string"
+        },
+        "streetAddress": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "email": {
+      "type": "string",
+      "format": "email"
+    },
+    "phoneNumber": {
```

Comment:
@decentralgabe Should we use `telephone` here rather than `phoneNumber`?

https://schema.org/telephone